src/parameterized_tendencies/microphysics/microphysics_wrappers.jl

```diff
@@ -324,3 +325,182 @@ function compute_precipitation_sinks!(
     @. Sqₛᵖ += Sᵖ
     #! format: on
 end
+
+#####
+##### 2M microphysics
+#####
+
+"""
+    aerosol_activation_sources(cm_params, thp, ts, dt)
+
+ - cm_params - CloudMicrophysics parameters struct for cloud water or ice condensate
+ - thp - Thermodynamics parameters struct
+ - ts - thermodynamics state
+ - qₚ - precipitation (rain or snow) specific humidity
+ - n_dp - number concentration droplets (liquid or ice) per mass
+ _ n_dp_prescribed - prescribed number concentration of droplets (liquid or ice) per mass
+ - dt - model time step
+
+Returns the activation rate. #TODO This function temporarily computes activation rate 
+based on mass rates and a prescribed droplet mass (no activation parameterization yet).
+"""
+function aerosol_activation_sources(
+    cm_params::Union{CMP.CloudLiquid{FT}, CMP.CloudIce{FT}},
+    thp,
+    ts,
+    qₚ,
+    n_dp,
+    n_dp_prescribed,
+    dt,
+) where {FT}
+    r_dp = FT(2e-6) # 2 μm
+    m_dp = 4 / 3 * π * r_dp^3 * 1000 # ρ_water = 1000 kg/m^3
```

Comment:
We should grab water density from parameters. I think it's already part of some parameters struct. Maybe even in 2M scheme if we are lucky. And if not, we can quickly add it there